velox/functions/prestosql/Bitwise.h

```diff
@@ -15,10 +15,12 @@
  */
 #pragma once
 
+#include <bitset>
```

Comment:
I wonder if something like countBits from velox/common/base/BitUtil.h could be used instead.